docs/3.0.0-beta.x/concepts/models.md

```diff
@@ -218,7 +218,17 @@ To improve the Developer Experience when developing or using the administration
 
 ### Exceptions
 
-- `uid` — This field type allows a `targetField` key. The value is the name of an attribute thas has `string` of `text` type.
+**`uid`**
+
+This field type allows a `targetField` key. The value is the name of an attribute that has `string` or `text` type.
+
+The field type also allows an `options` key. The value is an `object` of options passed to [the `uid` generator](https://github.com/sindresorhus/slugify).
+
+- `separator` (string) — Specify the string to be used for word separation. Allowed separators are: `"-"`, `"_"`, `"."`, and `"~"`. Default: `"-"`
```

Comment:
Can you remove the parameters from our documentation so we don't have to maintain it and simply redirect to the lib doc ?

Reply:
I added it because the options allowed by the slugify options doesn't match the the uid validation regex. The validation regex only allows for `"-"`, `"_"`, `"."`, and `"~"`, where slugify allows for any string.

I could replace the docs for the options with a note stating the uid generated must conform to the validation regex?

Reply:
Yeah I would link to the docs of the lib and add a warning to explain the constraints

Reply:
I've made some changes to the documentation, changing what we discussed